src/app/shared/markdown-wrapper/markdown-wrapper.component.html

```diff
@@ -0,0 +1 @@
+<div fxFlex [innerHTML]="customSanitize(data)"></div>
```

Comment:
A risk of using a function like this in HTML is that it gets a lot. Can you verify that it's not? If it is called a ton, then you need to have a field that you update whenever the data changes.

Reply:
Good catch. Now passing in a class variable. This variable is updated when the component's input variable is modified, meaning we only call the sanitization function on input change.